### PR TITLE
Update git / workflow / staging-ii

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Change Link Name

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/git/workflow/staging-ii/untrack-a-directory-file-without-deleting-it.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/git/workflow/staging-ii/untrack-a-directory-file-without-deleting-it.md?ref=34ee5277715eff0ed1bd6048b1c99a806551e53e)

File: git/workflow/staging-ii/create-a-new-branch-from-a-stash.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](http://git-scm.com/book/en/v2/Git-Tools-Stashing-and-Cleaning){website}'
+  - '[Stashing And Cleaning Official Documentation](http://git-scm.com/book/en/v2/Git-Tools-Stashing-and-Cleaning){website}'
```
Comment (kapnobatai137) [inline git/workflow/staging-ii/create-a-new-branch-from-a-stash.md]:
```suggestion
  - '[Stashing And Cleaning](http://git-scm.com/book/en/v2/Git-Tools-Stashing-and-Cleaning){documentation}'
```

File: git/workflow/staging-ii/delete-all-untracked-files.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](https://git-scm.com/docs/git-clean){website}'
+  - '[Git Clean Official Documentation](https://git-scm.com/docs/git-clean){website}'
```
Comment (kapnobatai137) [inline git/workflow/staging-ii/delete-all-untracked-files.md]:
```suggestion
  - '[Git Clean](https://git-scm.com/docs/git-clean){documentation}'
```

File: git/workflow/staging-ii/stashing-changes.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](http://git-scm.com/book/en/v2/Git-Tools-Stashing-and-Cleaning){website}'
+  - '[Stashing And Cleaning Official Documentation](http://git-scm.com/book/en/v2/Git-Tools-Stashing-and-Cleaning){website}'
```
Comment (kapnobatai137) [inline git/workflow/staging-ii/stashing-changes.md]:
```suggestion
  - '[Stashing And Cleaning](http://git-scm.com/book/en/v2/Git-Tools-Stashing-and-Cleaning){documentation}'
```

File: git/workflow/staging-ii/untrack-a-directory-file-without-deleting-it.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ aspects:
 
 links:
 
-  - '[github.com](https://github.com/jbranchaud/til/blob/master/git/untrack-a-file-without-deleting-it.md){website}'
+  - '[Untrack A File Without Deleting It github.com](https://github.com/jbranchaud/til/blob/master/git/untrack-a-file-without-deleting-it.md){website}'
```
Comment (kapnobatai137) [inline git/workflow/staging-ii/untrack-a-directory-file-without-deleting-it.md]:
```suggestion
  - '[Untrack A File Without Deleting It](https://github.com/jbranchaud/til/blob/master/git/untrack-a-file-without-deleting-it.md){website}'
```

File: git/workflow/staging-ii/useful-stashing-options.md [hunk 5]
```diff
@@ -24,7 +24,7 @@ parent: stashing-changes
 links:
 
   - >-
-    [git-scm.com](http://git-scm.com/book/en/v2/Git-Tools-Stashing-and-Cleaning){website}
+    [Stashing And Cleaning Official Documentation](http://git-scm.com/book/en/v2/Git-Tools-Stashing-and-Cleaning){website}
```
Comment (kapnobatai137) [inline git/workflow/staging-ii/useful-stashing-options.md]:
```suggestion
    [Stashing And Cleaning](http://git-scm.com/book/en/v2/Git-Tools-Stashing-and-Cleaning){documentation}
```

**Pull request merged**